CHANGELOG.rst

```diff
@@ -11,6 +11,10 @@ Unreleased
   ``dependson`` is now a list instead of a filter iterator. This fixes issue #72 
   where dependencies were only loaded in the first bulk load.
 
+  **Changed**
+  SQLSource now has a ``fetch_size`` constructor parameter such the end-user can
+  control how much data should be held in main memory for each round trip to the SQLSource.
```

Comment:
Maybe replace SQLSource with RDBMS or source (for consistency with the doc strings) as SQLSource is the class.